repositories/like_repository.py

```
@@ -0,0 +1,63 @@
+import json
+import os
+from typing import List, Optional
+
+from common.config import settings
+
+
+class LikeRepository:
+    def __init__(self):
```

Ruff (0.14.13)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)